data/mods/Magiclysm/traits/manatouched.json

```diff
@@ -0,0 +1,134 @@
+[
+  {
+    "type": "mutation_category",
+    "id": "MANATOUCHED",
+    "name": "Manatouched",
+    "threshold_mut": "THRESH_MANA",
+    "mutagen_message": "Your mana leylines surge.",
+    "iv_message": "A crackle of power comes from within.",
+    "iv_sound": true,
+    "iv_sound_message": "a crackle",
+    "iv_sound_id": "crackle",
+    "iv_noise": 1,
+    "iv_additional_mutations": 0,
+    "memorial_message": "Ascended to the metaphysical."
+  },
+  {
+    "type": "mutation",
+    "id": "THRESH_MANA",
+    "name": { "str": "Manatouched" },
+    "points": 1,
+    "description": "Magic is in you.  Magic is you.",
+    "mana_multiplier": 1.5,
+    "valid": false,
+    "purifiable": false,
+    "threshold": true
+  },
+  {
+    "type": "mutation",
+    "id": "MANA_CRYST_MANA",
+    "name": { "str": "Mana Solidification" },
+    "points": 1,
+    "category": [ "MANATOUCHED" ],
+    "description": "Your mana leylines have become much easier for you to control, resulting in you being able to naturally create crystallized mana.",
+    "spells_learned": [ [ "manatouched_crystal", 5 ] ]
```

Comment:
If I purify this away, do I lose the spell?

Reply:
yes, you should.